lms/templates/edxnotes/note-item.underscore

```diff
@@ -37,7 +37,12 @@
 
         <% if (tags.length > 0) { %>
             <h3 class="reference-title"><%- gettext("Tags:") %></h3>
-            <span class="reference-meta reference-tags"><%- tags.join(", ") %></span>
+            <% for (var i = 0; i < tags.length; i++) { %>
+                <a class="reference-meta reference-tags" href="#"><%- tags[i] %></a>
+                <% if (i < tags.length - 1) { %>
+                    <span class="reference-meta reference-tags">,</span>
```

Comment:
I would remove this span, and instead add a css style to add in the comma.  (content: ","; color: $gray ) This will resolve the feedback about the spacing between the tag and the commas.

The selector you would be adding is this one: 
a.reference-meta.reference-tags:after